jenkins/Jenkinsfile

```diff
@@ -27,7 +27,7 @@ pipeline {
           }
           steps {
             sh '''
-              julia --project=monorepo -e 'using Pkg; Pkg.Registry.update(); Pkg.develop(; path="./NDTensors"); Pkg.develop(; path="."); Pkg.develop(; path="./ITensorGPU"); Pkg.test("ITensorGPU");'
+              julia -e 'using Pkg; Pkg.Registry.update(); Pkg.develop(path="./NDTensors"); Pkg.develop(; path="./ITensorGPU"); Pkg.add("CUDA"); using CUDA; CUDA.set_runtime_version!("v11.8"); Pkg.test("ITensorGPU");'
```

Comment:
This new version seems to drop testing against the development version of `ITensors` (the operation `Pkg.develop(; path=".")`), I think we should keep that.

Reply:
Okay, I added it back!